tanglescope/common/iri.hpp

```diff
@@ -174,6 +174,22 @@ class LMHSMessage : public IRIMessage {
   std::string _latestSolidMilestoneHash;
 };
 
+class LMSIMessage : public IRIMessage {
+ public:
+  explicit LMSIMessage(std::string_view from) {
+    auto chunks = IRIMessage::chunks(from);
+    this->_latestSolidMilestoneIndex = std::stoull(std::move(chunks[1]));
+  }
+
+  inline IRIMessageType type() { return IRIMessageType::LMSI; }
+
+ public:
+  const uint64_t& latestSolidMilestoneIndex() { return this->_latestSolidMilestoneIndex; }
```

Comment:
shouldn't return by reference here. a reference is just as large as the actual value anyway.

Reply:
was overlooking, thanks